app/assets/locales/locale-en.json

```diff
@@ -1611,7 +1611,7 @@
             "The current node is out of sync with the blockchain, try switching to another one",
         "sync_yes": "The current node is in sync with the blockchain",
         "target_collateral_ratio":
-            "Setting a target collateral ratio can help with not selling all of the posision at once.<br /><br /><strong>Sell as little by setting target below MCR (ex. 1.5)</strong><br /><br /><strong>Sell as little but decrease possible margin calls again by setting it higher than MCR (ex. 3)",
+            "By setting a Target Collateral Ratio (TCR) a soft limit can be set when margin called.<br /><br />To sell small portions more ofthen, set target below MCR (ex. 1.5)<br /><br />To sell larger portions less ofthen, set target abow MCR (ex. 3)",
```

Comment:
This has several typos, ofthen > often, abow > above. I also find it a little confusing, especially the first sentence.

How about this: "Setting a Target Collateral Ratio (TCR) will allow the blockchain to automatically sell the required amount of your collateral in order to keep your collateral ratio at or above your defined TCR when the value of your collateral decreases."

Reply:
Sorry for the typos, appreciate your help with catching them.

Your suggested sentence seems much better. I tried to find a good explanation for the common user to understand what it did.